contracts/sol6/KyberNetwork.sol

```diff
@@ -1112,12 +1112,14 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils5, IKyberNetwork, Reentra
             }
             totalBps += reservesData.splitsBps[i];
 
-            destAmount += calcDstQty(
+            uint256 destQty = calcDstQty(
```

Comment:
lets use destAmount and totalDestAmount
so we are consistent.

in general lets use amount and not qty in all new things we add.

maybe later replace old ones as well.

Reply:
Updated